persistence/mongodb/src/main/scala/net/liftweb/mongodb/Mongo.scala

```diff
@@ -126,6 +127,7 @@ object MongoDB {
     * and the use of getLastError.
     * See: http://docs.mongodb.org/ecosystem/drivers/java-concurrency/
     */
+  @deprecated("No longer relevant. See mongo-java-drivers's JavaDocs for details", "3.0")
```

Comment:
Correct. `Acknowledged` writes are now the preferred way to do this and so Mongo is deprecating this functionality.

"No longer needed" is probably better. I'll update it.